Eshop.Persistence/EshopDbContext.cs

```diff
@@ -14,10 +16,14 @@ public EshopDbContext(DbContextOptions<EshopDbContext> options) : base(options)
         }
 
         public DbSet<Category> Categories  { get; private set; }
+        public DbSet<Product> Products { get; private set; } 
 
-        protected override void OnModelCreating(ModelBuilder builder)
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
-            builder.ApplyConfiguration(new CategoryConfiguration());
+            modelBuilder.ApplyConfiguration(new CategoryConfiguration());
+            modelBuilder.ApplyConfiguration(new ProductConfiguration());
+
+            base.OnModelCreating(modelBuilder);
```

Comment:
this was not needed for Category, so there is no need use it for Product

Reply:
not sure if you mean 
modelBuilder.ApplyConfiguration(new CategoryConfiguration());
modelBuilder.ApplyConfiguration(new ProductConfiguration());
but removed